make/ex_doc_wrapper.erl

```diff
@@ -0,0 +1,54 @@
+-module(ex_doc_wrapper).
+
+-export([main/1]).
+
+main(Args) ->
+    case os:find_executable("ex_doc") of
+        false ->  
+            case prompt() of
+                {ok, download} ->
+                    case download_otp() of
+                        error ->
+                            io:format("Unable to read ERL_TOP from environment?~n");
+                        ok ->
+                            main(Args)
+                    end;
+                {error, try_again} ->
+                    main(Args);
+                {error, do_not_download} ->
+                    io:format("Skipping download...~n")
+            end;
+
+        ExDocExe ->
+            Cmd = lists:append(["exec ", ExDocExe, " "],  lists:join(" ", Args)),
+            io:format("Running ~s~n", [Cmd]),
+            os:cmd(Cmd)
```

Comment:
I tried the same

```
  Out = os:cmd(Cmd),
  io:format("~s", [Out])
```

here, but I got some kind of binary output back from `ex_doc` so I didn't worry about it.

Reply:
I think it has to do with the output being colorized